SD-K9/src/app/components/building-info/building-info.component.ts

```diff
@@ -100,6 +105,17 @@ export class BuildingInfoComponent implements OnInit, OnDestroy {
     return true;
   }
 
+  toggleBuildingInfoCardTabs(e){
```

Comment:
Is it possible to add the type of variable 'e'?
I think adding types help the code be more self-explanatory.